insurance/resources.ipynb

```diff
@@ -32,11 +38,31 @@
     "SECRET_KEY = os.getenv(\"AWS_SECRET_ACCESS_KEY\",\"\")\n",
     "\n",
     "## SQL\n",
+    "# DBHOSTNAME should be provided in the one of the following format.\n",
     "DBHOSTNAME = os.getenv(\"DBHOSTNAME\",\"\")\n",
     "DATABASENAME= os.getenv(\"DATABASENAME\",\"\")\n",
     "DBUSERNAME = os.getenv(\"DBUSERNAME\",\"\")\n",
     "DBPASSWORD = os.getenv(\"DBPASSWORD\",\"\")\n",
     "\n",
+    "# supported value are mysql or mssql\n",
+    "DB_PROVIDER = \"mysql\"\n",
+    "\n",
+    "# dataset to be used as training data\n",
+    "DKUBE_BASE_DATASET = \"insurance-data\"\n",
+    "if DATA_SOURCE == \"sql\":\n",
+    "    DKUBE_BASE_DATASET = f\"{DKUBE_BASE_DATASET}-sql\"\n",
+    "\n",
+    "\n",
+    "# Model name to be created or used for example, it will create the model\n",
+    "# if not existing otherwise it will ignore creation.\n",
+    "MODEL_NAME = 'insurance-model'\n",
+    "\n",
+    "\n",
+    "# If you  are doing retraining then you need to provide it\n",
+    "RETRAINING_DATASET = 'insurance-training-data'\n",
```

Comment:
For sql, you have a different training dataset. But, you don't have a different retraining dataset?

Reply:
okay i will check this, if earlier it was handled correctly it will work

Reply:
@pallavi-pannu-oc are we doing retraining unconditionally with local dataset? i see in original code also it is using retraining dataset

Reply:
@gaurav-oc in preprocessing stage we are picking up the retraining or training data depending on user and then doing training or retraining in the next step.. even though name is insurance-training-data, but the content of it will be taken care by preprocessing step..